app/services/source-filters.ts

```diff
@@ -82,6 +94,15 @@ class SourceFiltersViews extends ViewHandler<{}> {
     ];
   }
 
+  get presetFilterOptionsReact() {
+    return this.presetFilterOptions.map(opt => ({
+      label: opt.title,
+      // Empty string doesn't play nicely with our react list input
+      // due to it being a falsey value in JS.
+      value: opt.value === '' ? 'none' : opt.value,
+    }));
+  }
```

Comment:
is there a reason to make this an additional mapping function vs just changing `presetFilterOptions`? are we still using that in vue somewhere?

Reply:
Yeah, the hardware onboarding is still Vue, and uses the preset values.